aws/table_aws_kinesisanalyticsv2_application.go

```diff
@@ -0,0 +1,253 @@
+	}
+	plugin.Logger(ctx).Trace("listKinesisAnalyticsV2Applications", "AWS_REGION", region)
+
+	// Create session
+	svc, err := KinesisAnalyticsV2Service(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	// List call
+	resp, err := svc.ListApplications(&kinesisanalyticsv2.ListApplicationsInput{})
+	for _, application := range resp.ApplicationSummaries {
+		d.StreamListItem(ctx, application)
+	}
+
+	return nil, err
+}
+
+//// HYDRATE FUNCTIONS
+
+func getKinesisAnalyticsV2Application(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	logger.Trace("getKinesisAnalyticsV2Application")
+
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+
+	var applicationName string
+	if h.Item != nil {
+		i := h.Item.(*kinesisanalyticsv2.ApplicationSummary)
+		applicationName = *i.ApplicationName
+	} else {
+		applicationName = d.KeyColumnQuals["application_name"].GetStringValue()
+	}
+
+	// Create Session
+	svc, err := KinesisAnalyticsV2Service(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	// Build the params
+	params := &kinesisanalyticsv2.DescribeApplicationInput{
+		ApplicationName: &applicationName,
+	}
+
+	// Get call
+	data, err := svc.DescribeApplication(params)
+	if err != nil {
+		logger.Debug("getKinesisAnalyticsV2Application", "ERROR", err)
+		return nil, err
+	}
+
+	return data.ApplicationDetail, nil
+}
+
+func getKinesisAnalyticsV2ApplicationTags(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	logger.Trace("getKinesisAnalyticsV2ApplicationTags")
+
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+	arn := getApplicationArn(h.Item)
+
+	// Create Session
+	svc, err := KinesisAnalyticsV2Service(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	// Build the params
+	params := &kinesisanalyticsv2.ListTagsForResourceInput{
+		ResourceARN: arn,
+	}
+
+	// Get call
+	op, err := svc.ListTagsForResource(params)
+	if err != nil {
+		logger.Debug("getKinesisAnalyticsV2ApplicationTags", "ERROR", err)
+		return nil, err
+	}
+
+	return op, nil
+}
+
+//// TRANSFORM FUNCTIONS
+
+func kinesisAnalyticsV2ApplicationTagListToTurbotTags(ctx context.Context, d *transform.TransformData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("route53resolverTagListToTurbotTags")
+
+	if d.Value == nil {
+		return nil, nil
+	}
+	tagList := d.Value.([]*kinesisanalyticsv2.Tag)
+
+	// Mapping the resource tags inside turbotTags
+	var turbotTagsMap map[string]string
+	if tagList != nil {
+		turbotTagsMap = map[string]string{}
+		for _, i := range tagList {
+			turbotTagsMap[*i.Key] = *i.Value
+		}
+	} else {
+		return nil, nil
+	}
+
+	return turbotTagsMap, nil
+}
+
+func getApplicationArn(item interface{}) *string {
```

Comment:
Don't use `get` prefix in function name other than hydrate functions.
```suggestion
func applicationArn(item interface{}) *string {
```